### DISCUSSION TITLE
Сверка схем АРС-Еж3, подумать над подтверждением бдительности от ПБ

### DISCUSSION THREAD

**Discussion by fixinit75** [metrostroi-repo/MetrostroiAddon#205] [General]
Из поездной автоматики
![image](https://user-images.githubusercontent.com/78933401/115115288-4e29aa00-9f9c-11eb-9444-4753f5d13c7b.png)

**Comment by glebqip** [comment]
Какое сейчас поведение АРС в игре при:
Нажатии педали при наличии частоты
Не нажатии педали при проскакивании ОЧ
Превышении без нажатия КВТ

**Comment by fixinit75** [comment]
Сейчас педаль частично запараллелена с КВТ.
При превышении без нажатия КВТ происходит автоматическое включение тормозного режима до снижения скорости до разрешённой, тормозной режим отключается, включается таймер, через опеределенное время, автоматически включается тормозной режим до полной остановки или до подтверждения бдительности.

При проскакивании ОЧ и при не нажатии педали происходит автоматическое включение тормозного режима и его моментальное выклчение, через опеределенное время, автоматически включается тормозной режим до полной остановки или до подтверждения бдительности.

При наличии частоты, нажатие педали на время, чуть меньше одной секунды ничего не дает, а при длительном нажатии, включат тормозной режим до отпускания ПБ.

И сейчас нажатие педали дает команду на подтверждение бдительности

Так быть не должно. ПБ не подтверждает бдительность в ЕЖ3, принцип ее работы должен быть приблизительно такой же, как и в 81-720. 
Прилагаю поездную автоматику, читать со страницы 29 
[https://docplayer.ru/67249249-Sistemy-ars-i-poezdnaya-avtomatika.html](url)

**Comment by glebqip** [comment]
Да, то что ПБ отменяет вроде как неверно. Но доки такие сюда не очень подходят.
Надо сверить схемы АРС ещё раз по выхинской книжке и по потерянной странице из неё
Тут по потерянной странице может сказать @toxin1pilot

**Comment by alexandr-sizov** [comment]
Все в этой части сделано по Выхинской книге. Есть один момент который постараюсь уточнить. Мб в нем прикол с подтверждением от ПБ и есть. УПЦшным говнокнижкам я не доверяю, к слову даже по описаню из нее про включение АРС при 0 или ОЧ, можно сделать вывод что ПБ так же подтверждает бдительность. Так как если бы паралели в этой части не было, то после включения АРС нужно было бы нажать КВТ, а уж только потом жать ПБ и начинать движение.

Полной проверкой схем АРС-ЕЖ3 по имеющимся схемам я займусь в ближайшее время.

**Comment by toxin1pilot** [comment]
На данный момент схемы Еж3 соответствуют реальным на 99%. Логика работы АРС бралась из книжки Выхинской, поскольку это единственный источник, которому можно доверять. УПЦшные и прочие конспекты лично я ни во что не ставлю, у них АРС в Еж3 и 2/6 и 2/5 и как только не называется. Потому считаю эту тему исчерпанной полностью.

**Comment by alexandr-sizov** [comment]
![image](https://user-images.githubusercontent.com/31768135/115999810-ea375f00-a5f5-11eb-99d9-0238c3b27783.png)
[18:30] Αλέξανδρος: Есть вариант, по схеме, что может быть, через БРки РНТ под ток встать не может, а только удержать якорь, а встать под ток только от КВТ
 Αλέξανδρος: и мне очень не нравится левая цепь
[18:32] Αλέξανδρος: 1. там точно не 12 вольт получается на реле
[18:32] Αλέξανδρος: 2. вопрос сколько
[18:33] Αλέξανδρος: с учетом R28
[18:33] Αλέξανδρος: и с учетом посадки ее на землю через R35

**Comment by alexandr-sizov** [comment]
По информации от машинистов: де факто подтверждение только от КВТ, что косвенно подтверждается детальным разбором схемы РНТ и консультацией со специалистом по электротехнике. 


Посовещавшись с Глебом, принято решение убрать подтверждение от ПБ.
